File: Week-04/Day-1/Sharpie.py
class Sharpie(object):

    def __init__(self, color, width):
        self.color = str(color)
        self.width = float(width)
        self.ink_amount = 100
    
    def use(self):
        self.ink_amount -= 1

red_sharpie = Sharpie("red", 11)
red_sharpie.use()

print(red_sharpie.ink_amount, red_sharpie.color)

File: week-08/day-1/sharpie.js
class Sharpie {

    constructor(color, width) {
        this.color = color;
        this.width = width;
        this.inkAmount = 100;
    }

    use() {
        this.inkAmount -= this.width;
    }
}

var greenSharpie = new Sharpie('green', 10);

while (greenSharpie.inkAmount >= greenSharpie.width) {
    greenSharpie.use();
    console.log(greenSharpie.inkAmount);
}